src/lib/stores/busy.store.ts

```diff
@@ -0,0 +1,48 @@
+import { derived, writable, type Readable } from "svelte/store";
+import type { BusyState } from "$lib/types/busy";
+
+/**
+ * Store that reflects the app busy state.
+ * Is used to show the busy-screen that locks the UI.
+ */
+const initBusyStore = () => {
+  const { subscribe, update } = writable<Array<BusyState<any>>>([]);
```

Comment:
we should not use `any`. we can replace it with the usage of generics?

```
const initBusyStore = <T>() => {
  const { subscribe, update } = writable<Array<BusyState<T>>>([]);
    ...
```

Then 

export const busyStore = initBusyStore();